silverback/application.py

```diff
@@ -136,6 +139,9 @@ def __get_user_taskdata_handler(self, task_type: TaskType) -> list[TaskData]:
         assert str(task_type).startswith("user:"), "Can only fetch user task data"
         return self.tasks.get(task_type, [])
 
+    def __get_user_all_taskdata_handler(self) -> list[TaskData]:
```

Comment:
Can probably just return the full dict with this new method 

Originally made the other task segment by single task type so that in case we want dynamic registration there might be some measn of handling that (like say the app startup task registers a bunch of runtime tasks)

Reply:
The full dict isn't needed.  Task types are in labels so it's simpler to just iterate over them all and process them that way.